x-pack/plugins/ml/server/routes/modules.ts

```diff
@@ -36,16 +36,17 @@ function getModule(context: RequestHandlerContext, moduleId: string) {
 function saveModuleItems(
   context: RequestHandlerContext,
   moduleId: string,
-  prefix: string,
-  groups: string[],
-  indexPatternName: string,
+  prefix: string | undefined,
+  groups: string[] | undefined,
+  indexPatternName: string | undefined,
   query: any,
-  useDedicatedIndex: boolean,
-  startDatafeed: boolean,
-  start: number,
-  end: number,
+  useDedicatedIndex: boolean | undefined,
+  startDatafeed: boolean | undefined,
+  start: number | undefined,
+  end: number | undefined,
   jobOverrides: JobOverride[],
-  datafeedOverrides: DatafeedOverride[]
+  datafeedOverrides: DatafeedOverride[],
```

Comment:
nit, for consistency i think these `jobOverrides` and `datafeedOverrides` should be optional and be either a single or array of overrides

Reply:
https://github.com/elastic/kibana/pull/60656/commits/085960a936a2882f011597b75bc88831d504de73